revisions/N0/src/text/dialog_3.asm

```diff
@@ -1021,171 +1639,300 @@ Dialog178::
+;    db "someone to      "
+;    db "overcome the    "
+;    db "darkness. Find  "
+;    db "all the secret  "
+;    db "shells and go   "
+;    db "through the     "
+;    db "gate to receive "
+;    db "the ultimate    "
+;    db "sword!@"
+; Forslag til synonymer for «ultimate» tas imot med thx
 
 Dialog17A::
-    db "Hmmm. No        "
-    db "response. You   "
-    db "must not have   "
-    db "enough shells.@"
+    db "Hmmm. Ingen ting"
+    db "skjedde. Du har "
+    db "nok ikke nok    "
+    db "skjell.@"
+;    db "Hmmm. No        "
+;    db "response. You   "
+;    db "must not have   "
+;    db "enough shells.@"
 
 Dialog17B::
-    db "My job here     "
-    db "is finished.@"
+Dialog17B::
+    db "Min jobb her er "
+    db "fullført.@"
+;    db "My job here     "
+;    db "is finished.@"
 
 Dialog17C::
-    db "Hey you! Have   "
-    db "you been to the "
-    db "Camera Shop in  "
-    db "Tal Tal Heights?"
-    db "Iyoung people   "
-    db " like to go ther"
-    db "eand take       "
-    db " pictures of    "
-    db " themselves.    "
-    db " Seems kinda    "
-    db " funny to me.@"
+    db "Hei du! Har du  "
+    db "vært i foto-    "
+    db "butikken i Tipp-"
+    db "topphøydene?    "
+    db "Ungdommer liker "
+    db "å gå dit og ta  "
+    db "bilder av seg   "
+    db "selv. Virker    "
+    db "litt rart, spør "
+    db "du meg.@"
+;    db "Hey you! Have   "
+;    db "you been to the "
+;    db "Camera Shop in  "
+;    db "Tal Tal Heights?"
+;    db "Iyoung people   "
+;    db " like to go ther"
+;    db "eand take       "
+;    db " pictures of    "
+;    db " themselves.    "
+;    db " Seems kinda    "
+;    db " funny to me.@"
 
 Dialog17D::
-    db "Step right up   "
-    db "and get your    "
-    db "souvenir photo!@"
+    db "Kom inn og få en"
+    db "suvenir i form  "
+    db "av et bilde!@"
+;    db "Step right up   "
+;    db "and get your    "
+;    db "souvenir photo!@"
 
 Dialog17E::
-    db "Good job!       "
-    db "Use it on your  "
-    db "enemies and see "
-    db "what happens.   "
-    db "If you run out, "
-    db "go to the for-  "
-    db "est, pick some  "
-    db "mushrooms, and  "
-    db "I will make you "
-    db "more.@"
+    db "Godt jobba! Bruk"
+    db "det på fiendene "
+    db "dine og se hva  "
+    db "som skjer. Hvis "
+    db "du går tom, er  "
+    db "det bare å dra  "
+    db "til skogen og   "
+    db "plukke mer sopp,"
+    db "så skal jeg lage"
+    db "mer.@"
+;    db "Good job!       "
+;    db "Use it on your  "
+;    db "enemies and see "
+;    db "what happens.   "
+;    db "If you run out, "
+;    db "go to the for-  "
+;    db "est, pick some  "
+;    db "mushrooms, and  "
+;    db "I will make you "
+;    db "more.@"
 
 Dialog17F::
-    db ". . . .! I can't"
-    db "move! But I am  "
-    db "still all right."
-    db "Your little     "
-    db "sword won't     "
-    db "break this      "
-    db "bottle!@"
+   db ". . . . ! Jeg kan"
```

Comment:
```suggestion
    db ". . . .! Jeg kan"
```
her må et tegn fjernes. det er 17. foreslår siste punkt før ! for å gjøre det likt som på engelsk